docs/_articles/en/soql/soql-builder.md

```diff
@@ -111,20 +111,17 @@ Split your view to see your query in both SOQL Builder and the text editor.
 
 ![Split your screen to see both SOQL Builder and Text Editor](./images/soql-builder-split-panels.gif)
 
-## Switch Between SOQL Builder and Text Editor
-
-You can easily toggle between viewing your SOQL statements in SOQL Builder and the text editor.
+## Save Query Results
 
-![Click the Switch Between SOQL Builder and Text Editor icon to toggle views](./images/soql-toggle.png)
+<!-- ekapner, 2/25, update, moved section closer to graphic that shows the save icons  -->
 
-## Save Query Results
+Click one of the Save icons to save the query results in a `.csv` or `.json` file in the location of your choice. Include any paths to saved files in the `.gitignore` file so that you don’t deploy it to your org or include it in source control.
```

Comment:
The language here makes it sound like a requirement, like the use must update the `.gitignore` in order to use the save file feature. Is that intentional?

In my mind I thought of the call out to be more informational. 
> If you don't want to deploy the query results to your org or include it in source control, then you must add the path to any saved files or directory to your .gitignore.

Reply:
Or maybe "Click one of the Save icons to save the query results in a `.csv` or `.json` file in the directory of your choice. Remember to include the query results file or its directory in the `.gitignore` file to prevent it being deployed to your org or added to your repo." ?

Reply:
Good feedback, thanks @jonnyhork & @jshackell-sfdc